Error: The `form` element was not requested.
Error: Use `aria-describedby` to reference a group description.
Error: For usability, it is recommended to put the group description at the top of the component, just after the label.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
diff --git a/output/Checkbox Group with Accessible Description/1/a6bf9bc5-a788-40c4-b286-854b52a36668.html b/output/Checkbox Group with Accessible Description/1/a6bf9bc5-a788-40c4-b286-854b52a36668.html
index 24ad48e..46f9f90 100644
--- a/output/Checkbox Group with Accessible Description/1/a6bf9bc5-a788-40c4-b286-854b52a36668.html	
+++ b/output/Checkbox Group with Accessible Description/1/a6bf9bc5-a788-40c4-b286-854b52a36668.html	
@@ -1,15 +1,16 @@
-<form>
-  <fieldset>
-    <legend>What fruits do you like?</legend>
-    <label>
-      <input type="checkbox" name="fruits" value="bananas"> Bananas
-    </label>
-    <label>
-      <input type="checkbox" name="fruits" value="oranges"> Oranges
-    </label>
-    <label>
-      <input type="checkbox" name="fruits" value="apples"> Apples
-    </label>
-    <small>You can change this later</small>
-  </fieldset>
-</form>
\ No newline at end of file
+<fieldset aria-describedby="fruit-desc">
+  <legend>What fruits do you like?</legend>
+  <small id="fruit-desc">You can change this later</small>
+  <label for="bananas">
+    <input type="checkbox" name="fruits[]" id="bananas" value="bananas">
+    Bananas
+  </label>
+  <label for="oranges">
+    <input type="checkbox" name="fruits[]" id="oranges" value="oranges">
+    Oranges
+  </label>
+  <label for="apples">
+    <input type="checkbox" name="fruits[]" id="apples" value="apples">
+    Apples
+  </label>
+</fieldset>
\ No newline at end of file
